app/views/merchant/items/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Item: <%= @item.name %></h1>
+<p>Description: <%= @item.description %></p>
+<p>Current selling price: <%= number_to_currency(@item.unit_price/100) %></p>
```

Comment:
I'd like to wait on a confirmation from Juliet or Abdul on this one if it truly breaks MVC, because I think this is a gray area. I'm manipulating data to display on the view page... Modification of data in the Model breaks MVC